ui/scripts/inlineHtml/inlineScripts.js

```diff
@@ -793,30 +797,123 @@ function updateUserStatistics() {
     });
   });
 }
-
-on('change:race', updateUserStatistics);
-
-// Update the abilities that are copied elsewhere on the character sheet
+// The change:repeating_skill:skillname catches new skills.
+on('sheet:opened change:race change:repeating_skill:skillname remove:repeating_skill', 
+   updateUserStatistics);
+
+function updateEffectiveness() {
+  getAttrs(['EP_t_max', 'SP_max', 'HP_max', 'best_weapon_damage', 'best_attack', 'best_attack_is_spell',
+            'current_dodge_with_armor', 'current_parry_with_armor', 'current_block_with_armor'], (attrs) => {
+    const attack = getNumberIfValid(attrs['best_attack']);
+    const defense = Math.max(getNumberIfValid(attrs['current_dodge_with_armor']),
+                             getNumberIfValid(attrs['current_parry_with_armor']),
+                             getNumberIfValid(attrs['current_block_with_armor']));
+    const EP = getNumberIfValid(attrs['EP_t_max']);
+    const SP = getNumberIfValid(attrs['SP_max']);
+    const HP = getNumberIfValid(attrs['HP_max']);
+    const attack_is_spell = getNumberIfValid(attrs['best_attack_is_spell']);
```

Comment:
I'll follow up and clean up the javaScript conventions to be standard camelCase.

Reply:
I just fixed the new variable underscored that I added, I think.